doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -7,8 +7,11 @@ OSSL_PROVIDER-default - OpenSSL default provider
 =head1 DESCRIPTION
 
 The OpenSSL default provider supplies the majority of OpenSSL's diverse
-algorithm implementations.  It also acts as a fallback when no other
-provider has been loaded.
+algorithm implementations. If an application doesn't specify anything else
+explicitly (e.g. in the application or via config), then this is the
+provider that will be used as fall-back: It is loaded automatically the
+first time that an algorithm is requested from a provider or a function
+acting on providers is called and no other provider has been loaded yet.
```

Comment:
There is some subtlety here.

We only attempt to load the fallback once. So if it is subsequently unloaded and no providers are available, we don't try again.

Also any attempt to explicitly load a provider (either via API or config) will disable subsequent fallback loading, even if the load attempt fails.

Reply:
OK - upon reading the Wiki, quite a lot of subtleties here... I'll try to amend....

Reply:
Done. Please check again.